js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -106,65 +128,117 @@ export const ResourceRoot: React.FC<Props> = (props) => {
-                                  </Tooltip>
-                                  {isDefault && <Tag>Default</Tag>}
-                                  {type === 'ENV_VAR' && <Tag intent="success">Env var</Tag>}
-                                </Box>
-                              </td>
+                    {Object.keys(nestedResources).length > 0 && (
+                      <Box>
+                        <SectionHeader>
+                          <Subheading>Resource dependencies</Subheading>
+                        </SectionHeader>
+                        <Table>
+                          <thead>
+                            <tr>
+                              <th style={{width: 120}}>Key</th>
+                              <th style={{width: 180}}>Resource</th>
                             </tr>
-                          );
-                        })}
-                      </tbody>
-                    </Table>
+                          </thead>
+                          <tbody>
+                            {Object.keys(nestedResources).map((key) => {
+                              return (
+                                <tr key={key}>
+                                  <td>
+                                    <Box
+                                      flex={{direction: 'column', gap: 4, alignItems: 'flex-start'}}
+                                    >
+                                      <strong>{key}</strong>
+                                    </Box>
+                                  </td>
+                                  <td colSpan={2}>
+                                    <Tag icon="resource">
+                                      {' '}
+                                      <Link
+                                        to={workspacePathFromAddress(
+                                          repoAddress,
+                                          `/resources/${nestedResources[key].name}`,
+                                        )}
+                                      >
+                                        {resourceDisplayName(nestedResources[key])}
+                                      </Link>
+                                    </Tag>
+                                  </td>
+                                </tr>
+                              );
+                            })}
+                          </tbody>
+                        </Table>
+                      </Box>
+                    )}
+                    <Box>
+                      <SectionHeader>
+                        <Subheading>Configuration</Subheading>
+                      </SectionHeader>
+                      <Table>
+                        <thead>
+                          <tr>
+                            <th style={{width: 120}}>Key</th>
+                            <th style={{width: 90}}>Type</th>
+                            <th style={{width: 90}}>Value</th>
```

Comment:
Are these widths necessary? In the screenshot I notice that the table doesn't take up the entire width of its container.

Reply:
Matter of preference more than anything, it means the columns nicely line up on subsequent tables & widths don't change when navigating from one resource to another which I found jarring

<img width="921" alt="Screen Shot 2023-03-13 at 1 33 33 PM" src="https://user-images.githubusercontent.com/10215173/224826000-3cc92d2a-d920-4389-91e4-c08baf8c0ec7.png">
<img width="919" alt="Screen Shot 2023-03-13 at 1 33 46 PM" src="https://user-images.githubusercontent.com/10215173/224826003-baefecfc-3f4e-423c-b68d-0338c4ebb75b.png">

Reply:
Ahh, I see. If just the "Resource" (Table A) and "Type" (Table B) columns need to align, you could just put a pixel width on the first column of each table, and let the browser do the rest.